### DISCUSSION TITLE
Screenshare

### DISCUSSION THREAD

**Discussion by alg0trader** [stoatchat/discussions#178] [Feature Suggestions]
Allow for screen sharing to users in a voice channel.

**Comment by TerminallyDied** [comment]
very important feature to me would love it in revolt

**Comment by 0x4A-0x41-0x4B** [comment]
This would allow a direct comparison with the likes of Discord, Skype, etc. . This seems, at least to me, like the most important feature.

**Reply by TerminallyDied** [reply]
and if they actually had linux audio share it would surpass discords screenshare

**Comment by HunnyBon** [comment]
My friend circle relies on screenshare on Discord for a lot of things we like to do. This would be a very good feature. Use whatever windows APIs there are for Windows for capturing audio, and use Pipewire for audio/video on Linux.

**Comment by IAmTomahawkx** [comment]
i know this is old but, superseded by #196
